### ISSUE TITLE
some confusion over use

### ISSUE THREAD

**Issue by bbolker** [r-hub/rhub-frontend#6]
I was excited to find the front-end open at https://builder.r-hub.io/ ... things were a little confusing though. (If everything I'm reporting here is known, please ignore/close this issue). Hope this helps.

- When I initially uploaded a tarball (`lme4_1.1-13.tar.gz`) and tried to verify my e-mail via Github I got (after clicking on the button to allow r-hub appropriate access to my github info) "error: cannot verify e-mail address".  (That might be because I have a different e-mail address in the `DESCRIPTION` file than the one that's associated with my Github account?)
- I then validated by e-mail code. That worked, but I didn't initially think so because after accepting the upload r-hub sent me to a page that proclaimed "Not Found" ([here](https://builder.r-hub.io/status/lme4_1.1-13.tar.gz-f566a97dc1a0a1f64f0517b02bb53a08)).
- warnings and errors:
```
WARNING
Missing link or links in documentation object 'confint.merMod.Rd':
  ���boot.ci���  [this occurs on my own machine too ...]
...
code for methods in class ���merPredD��� was not checked for suspicious field assignments (recommended package ���codetools��� not available?)
[and many more like this]
In utils::download.file(url, path, method = download_method(), quiet = quiet,  :
    URL 'https://api.github.com/repos/mangothecat/remotes/contents/.gitmodules?ref=master': status was '404 Not Found'`
 ...
  NOTE
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  The vignette engine knitr::rmarkdown is not available, because the rmarkdown package is not installed. Please install it.
Warning: running command 'kpsewhich framed.sty' had status 1
Warning in test_latex_pkg("framed", system.file("misc", "framed.sty", package = "knitr")) :
  unable to find LaTeX package 'framed'; will use a copy from knitr
Loading required package: Matrix
Quitting from lines 255-256 (lmer.Rnw) 
Error: processing vignette 'lmer.Rnw' failed with diagnostics:
there is no package called 'codetools'
```

**Comment by gaborcsardi** [comment]
> That might be because I have a different e-mail address in the DESCRIPTION file than the one that's associated with my Github account?)

Yes, we can only verify the address that you use on GitHub. I think you are right and this should be make clear.

> r-hub sent me to a page that proclaimed "Not Found"

Hmmm, interesting. I will try to reproduce this.

> warnings and errors

Thanks, this is very useful.

**Comment by gaborcsardi** [comment]
FYI: the "Not Found" is fixed now, my regexp for package names was incorrect and I did not allow a number at the end. :/

I'll keep this issue open until the other two issues are fixed.
